mobile/security.html

```diff
@@ -59,6 +61,10 @@ <h2>Technologies in progress</h2>
 
       <section class="featureset exploratory-work">
         <h2>Exploratory work</h2>
+        <div data-feature="Permissions">
+          <p>Different APIs use different mechanisms to grant permission to use or access an underlying powerful feature. To ease design of permission-related code, the <a data-featureid="permissions-request">Requesting permissions</a> and <a data-featureid="permissions-revoke">Relinquishing permissions</a> proposals extend the <a href="https://www.w3.org/TR/permissions/">Permissions API</a> to provide a uniform function for requesting and revoking permission to use powerful features.</p>
```

Comment:
Am I understanding correctly that the absence of `data-featureid` in Permissions API's `<a>` in this section is for preventing the spec appearing in the wrong section's table?

Reply:
That's correct!

If that happens many times, we could introduce a specific mechanism, such as a `data-ignore` flag to be able to reuse `data-featureid`.